### PR TITLE
Setup helm deployment for dev

### PR DESCRIPTION
**Pull request by CslBing**
#### Context

-

#### Changes proposed in this PR

-

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/services/IntegrationEventTopicService.kt
Comment (chiaramapellimt) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/services/IntegrationEventTopicService.kt:1]: I am sure it's right but .build().build() is weird

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/integration/services/IntegrationEventTest.kt
Comment (chiaramapellimt) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/integration/services/IntegrationEventTest.kt:1]: await.atMost(10, TimeUnit.SECONDS).untilAsserted 

love it

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/services/IntegrationEventTopicService.kt
Comment (CslBing) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/services/IntegrationEventTopicService.kt:1]: Build() in line 35 is for MessageAttributeValue.builder(), and Build() in line 36 is for PublishRequest.builder()

File: src/main/resources/application-localstack.yml
[the remark this answers is not in the document]
Comment (CslBing) [reply src/main/resources/application-localstack.yml:1]: updated to use queue name instead of queue arn

**Review by chiaramapellimt — APPROVED** [review]

**Review by popey2700 — APPROVED** [review]

**Pull request merged**